webview-ui/src/components/GetSolutionDropdown.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import {
+  Dropdown,
+  DropdownList,
+  DropdownGroup,
+  DropdownItem,
+  MenuToggle,
+  MenuToggleAction,
+} from "@patternfly/react-core";
+import { effortLevels, SolutionEffortLevel } from "@editor-extensions/shared";
+import { EnhancedIncident } from "@editor-extensions/shared";
+import { useExtensionStateContext } from "../context/ExtensionStateContext";
+import { getSolution } from "../hooks/actions";
+import { EllipsisVIcon, WrenchIcon } from "@patternfly/react-icons";
+
+type GetSolutionDropdownProps = {
+  incidents: EnhancedIncident[];
+  scope: "workspace" | "issue" | "in-between" | "incident";
+};
+
+const GetSolutionDropdown: React.FC<GetSolutionDropdownProps> = ({ incidents }) => {
+  if (!incidents || incidents.length === 0) {
+    console.log("Empty Incidents");
+    return null;
+  }
+
+  const [isOpen, setIsOpen] = useState(false);
+  const { state, dispatch } = useExtensionStateContext();
+  const onGetSolution = (incidents: EnhancedIncident[], effort: SolutionEffortLevel) => {
+    dispatch(getSolution(incidents, effort));
+  };
+
+  const isButtonDisabled =
+    state.isFetchingSolution || state.isAnalyzing || state.serverState !== "running";
+
+  const menuToggle = (
+    <MenuToggle
+      variant="primary"
+      size="sm"
```

Comment:
Adding this causes the button to fall out of vertical alignment with respect to other toolbar items. Remove or add custom styling to center the button vertically.

Reply:
#397 Created because the big button looks comical in the table.